Assets/Scripts/PlayerMovement.cs

```diff
@@ -29,6 +30,7 @@ public class PlayerMovement : MonoBehaviour
         handleMovement();
         if (Input.GetKeyDown("space")) handleGrab();
         if (Input.GetMouseButtonDown(0) && currentlyGrabbed) handleItemAction();
+        if (Input.GetKeyDown("e") && currentlyGrabbed) handleEat();
```

Comment:
Just an idea but wouldn't it make more sense to also trigger this on mouse click? Then mouse click serves as the 'use' action of whatever you are holding